DiscordBot.js

```diff
@@ -15,45 +15,216 @@ class DiscordBot {
+                                    "§lsco", 
+                                    "§ls",
+                                    ""
+                                ]
+                            ],
+                            /*"§ping -> should send back a Pong! \n" +
                             "§refresh -> forces the bot to refresh his files \n" +
-                            "§addChannel name id/this -> adds a channel to send into"
+                            "Channel Commands: \n" +
+                            "§listChannels/lsch -> lists all channels \n" +
+                            "§addChannel/ach name id/this -> adds given channel \n" +
+                            "§rmChannel/rmch id/this -> removes given channel \n" +
+                            "File Commands: \n" +
+                            "§resend filename -> sends the file again !work in progress! \n",*/
+                            messageColor
                         );
                         break; 
-                    case msg.content.startsWith(prefix + "ping"):
+//ping
+                    case msg.content == prefix + "ping":
                         msg.reply('Pong!');
-                        console.log("Pong!");
                         break; 
-                    case msg.content.startsWith(prefix + "refresh"):
-                        console.log("Testcycle started manually!")
-                        msg.reply("Testcycle started manually!");
+//refresh
+                    case msg.content == prefix + "refresh":
+                        console.log("Testcycle started manually!");
+                        this.sendMessage(msg.channel.id, undefined, "Testcycle started manually!", messageColor);
                         testCycle();
                         break;
-                    case msg.content.startsWith(prefix + "addChannel"):
-                        let content = msg.content.split(" "); //split in arguments
-                        if (content.length == 3) {
-                            //Get the Channel ID if it should be the one the message was sent in.
-                            if (content[2] == "this") {
-                                content[2] = msg.channel.id;
+
+//Channels
+//listChannels
+                    case msg.content == prefix + "listChannels" || msg.content == prefix + "lsch":
+                        respondContent = [[],[]];
+                        //Rearrange array to fit the Discord Fields
+                        for (const channel of channelsFile.channels) {
+                            respondContent[0].push(channel.name);
+                            respondContent[1].push(channel.id);
+                        }
+                        this.sendList(msg.channel.id, "List of all Channels", ["Name", "ID"], respondContent)
+                        break;
+//addChannel
+                    case msg.content.startsWith(prefix + "addChannel ") || msg.content.startsWith(prefix + "ach "):
```

Comment:
Finde ich in diesem Zustand zu gefährlich, keine Fehlerabfangung oder Eingabeüberprüfung macht ein extremes Sicherheitsrisiko. Erstmal sperren, solange das nicht gewehrleistet ist.